### PR TITLE
Add a nice 404 page

### PR DESCRIPTION
**Pull request by robertkowalski**
uses pong.js from David Albert:
https://github.com/davidbalbert/pong.js

happy gaming!

### CODE REVIEW

File: templates/basic-theme/404.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="text-center">
+  <h2>404! So sad that this page does exist :( How about a Pong match?</h2>
```
Comment (robin-drexler) [inline templates/basic-theme/404.hbs]: Shouldn't it read "So sad that this page does >not< exist"? :D

**Comment by robertkowalski** [comment]
I just learned about a better way of error handling at the end of http://hapijs.com/api#error-response

from the commit message: 

```
Continue with request flow when no error appeared
(`!response.isBoom`).

When an error appears an error Object is returned and the error-
view is rendered.
```

**Comment by robin-drexler** [comment]
Lgtm

**Comment by robertkowalski** [comment]
thanks robin

**Pull request merged**